__tests__/app.test.js

```diff
@@ -0,0 +1,33 @@
+const testData = require('../db/data/test-data')
+const app = require('../app')
+const db = require('../db/connection')
+const seed = require('../db/seeds/seed')
+const request = require('supertest')
+
+beforeEach(() => {
+    return seed(testData)
+})
+
+afterAll(() => {
+    return db.end();
+})
+
+describe('/api/topics', () => {
+    test('GET: responds with 200 and an array of topic objects with properties of slug and description', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({body}) => {
+            expect(body.topics).toEqual([{

```

Comment:
this works for now but what if we have an array that is too large to type out or we don't know what each topic is going to look like 🙂. Can we refactor this to be dynamic and we can just test that each object contains the the keys we want to be in there.